var/spack/repos/builtin/packages/py-grapheme/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyGrapheme(PythonPackage):
+    """A Python package for working with user perceived characters. More
+    specifically, string manipulation and calculation functions for working
+    with grapheme cluster groups (graphemes) as defined by the
+    Unicode Standard Annex #29."""
+
+    homepage = "https://github.com/alvinlindstam/grapheme"
+    pypi = "grapheme/grapheme-0.6.0.tar.gz"
+
+    version("0.6.0", sha256="44c2b9f21bbe77cfb05835fec230bd435954275267fea1858013b102f8603cca")
+
+    depends_on("python@3.6:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@3.6:", type=("build", "run"))
```
Where do you see 3.6+?

Reply:
oh, that's a typo, I think I meant 3.5. The setup.py lists 3.5, 3.6, 3.7, and 3.8

Reply:
I generally don't specify unless the packager uses `python_requires` since pip doesn't read that metadata, but up to you if you want to do 3.5+.

Reply:
I'm fine with whatever is more correct for spack. The larger package that this is a dependency for requires 3.8 anyway so it doesn't affect me personally

Reply:
I would just leave it out then, Spack won't even support Python 3.6 and older after the next release anyway.